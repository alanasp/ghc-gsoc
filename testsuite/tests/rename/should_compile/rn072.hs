{-# OPTIONS_GHC -fwarn-warnings-deprecations #-}

-- Test deprecation of variable and data constructor exports

module ShouldCompile where

import Rn072_A

foo = bfunc
