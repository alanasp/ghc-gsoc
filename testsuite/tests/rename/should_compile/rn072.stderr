rn072.hs:9:7: warning: [-Wdeprecations (in -Wdefault)]
    In the use of ‘bfunc’
    (imported from Rn072_A, but defined in Rn072_B):
    Deprecated: "bfunc export from Rn072_B is deprecated"
